snuba/web/rpc/v1/resolvers/R_eap_spans/resolver_get_trace.py

```diff
@@ -55,19 +57,59 @@
         "attr_num",
         "span_id",
         "timestamp",
+        "time",
     ]
 ]
 
+NORMALIZED_COLUMNS_TO_INCLUDE_EAP_ITEMS = [
+    "organization_id",
+    "project_id",
+    "trace_id",
+    "sampling_weight",
+]
+
+# Map of eap_items attributes to eap_spans attributes to preserve backwards compatibility
+EAP_ITEMS_ATTRIBUTE_MAP = {
+    "sentry.raw_description": "name",
+    "sentry.transaction": "segment_name",
+    "sentry.start_timestamp_precise": "start_timestamp",
+    "sentry.end_timestamp_precise": "end_timestamp",
+    "sentry.duration_ms": "duration_ms",
+    "sentry.event_id": "event_id",
+    "sentry.exclusive_time_ms": "exclusive_time_ms",
+    "sentry.is_segment": "is_segment",
+    "sentry.parent_span_id": "parent_span_id",
+    "sentry.profile_id": "profile_id",
+    "sentry.received": "received",
+    "sentry.segment_id": "segment_id",
+}
+
+# Attributes that we store in eap_items but not eap_spans
+ATTRIBUTES_TO_SKIP_FROM_EAP_ITEMS = [
+    "event_id",
+    "received",
+]
+
 
 def _build_query(request: GetTraceRequest) -> Query:
     selected_columns: list[SelectedExpression] = [
         SelectedExpression(
             name="id",
-            expression=column("span_id", alias="id"),
+            expression=attribute_key_to_expression_eap_items(
+                AttributeKey(name="sentry.span_id", type=AttributeKey.Type.TYPE_STRING)
+            )
+            if use_eap_items_table(request.meta)
+            else column("span_id", alias="id"),
         ),
         SelectedExpression(
             name="timestamp",
-            expression=column(
+            expression=f.CAST(
```

Comment:
What is the difference between `f.CAST()` and `f.cast()`? We have both usages scattered in this file. Probably better to standardize on `f.cast()`.